src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -433,11 +472,3 @@
         display: none;
     }
 }
-
-/***** Media query print: Do not print the Toolbar. *****/
```

Comment:
I can't imagine anyone printing a web page in the `dev` environment. But if someone ever did, don't you think that they might actually want to print the entire page, including the toolbar?